client/shared/src/api/client/types/textDocument.test.ts

```diff
@@ -1,6 +1,5 @@
-import { DocumentSelector, TextDocument } from 'sourcegraph'
-
 import { Position } from '@sourcegraph/extension-api-types'
+import { DocumentSelector, TextDocument } from '@sourcegraph/shared/src/codeintel/legacy-extensions/api'
```

Comment:
`@sourcegraph/shared` imports used in the `shared` package create circular dependency graphs for some build tools (e.g., Bazel). With the current configuration, we need to use relative paths for internal package imports to avoid such issues. 
```suggestion
import { DocumentSelector, TextDocument } from '../../../codeintel/legacy-extensions/api'
```

Reply:
Thanks, will fix all of these.